rust/frontend/src/server.rs

```diff
@@ -1865,13 +1872,15 @@ async fn collection_get(
         include = ?payload.include,
         has_where = parsed_where.is_some(),
     );
+
     let request = GetRequest::try_new(
         tenant,
         database,
         collection_id,
-        payload.ids,
-        parsed_where,
-        payload.limit,
+        validated_payload.ids.map(|ids| ids.to_vec()),
```

Comment:
Before we didn't have to clone, now we do, that's not preferable - why this change?